scripts/src/main/resources/vscode-workspaces_vs.bat

```diff
@@ -0,0 +1,5 @@
+@echo off
+pushd %~dp0
+set WORKSPACE=workspaces_vs
+call scripts\vscode-start.bat
+popd
```

Comment:
I do not understand this script. For Eclipse we create a start script for each workspace. Is there only one workspace in the folder `workspace_vs`? So not multiple subfolders each as individual workspace?
If so, I got it and all is fine. Otherwise I would expect the scripts to be created per subfolder of `workspace_vs`.

According to the above code this still does not make sense to me:
> for /f "delims=" %%i in ('dir /a:d /b "%WORKSPACES_VS%\*.*"') do call create-or-update-workspace-vs.bat %%i noPause

This IMHO means that for each subfolder a separate workspace is created.